api/src/exam-environment/schemas/exam-attempt.ts

```diff
@@ -29,3 +29,42 @@ export const examEnvironmentPostExamAttempt = {
     500: STANDARD_ERROR
   }
 };
+
+const examEnvAttempt = Type.Object({
+  startTimeInMS: Type.Number(),
+  questionSets: Type.Array(
+    Type.Object({
+      id: Type.String({ format: 'objectid' }),
+      questions: Type.Array(
+        Type.Object({
+          id: Type.String({ format: 'objectid' }),
+          answers: Type.Array(Type.String()),
+          submissionTimeInMS: Type.Number()
+        })
+      )
+    })
+  ),
+  result: Type.Union([
+    Type.Null(),
+    Type.Object({
+      score: Type.Number(),
+      passingPercent: Type.Number()
+    })
+  ])
+});
+
+export const examEnvironmentGetExamAttempts = {
+  params: Type.Object({
+    attemptId: Type.Optional(Type.String({ format: 'objectid' }))
```

Comment:
```suggestion
    attemptId: Type.Union([
      Type.String({ format: 'objectid' }),
      Type.String({ maxLength: 0 })
    ])
```
Try this. It should behave itself.